docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,298 @@
+![DynamoDB Self-Hosted](../../../img/database-access/guides/dynamodb_selfhosted.png)
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A running Teleport cluster, either self hosted or in Teleport Cloud.
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- A host, e.g., an EC2 instance, where you will run the Teleport Application
+  Service.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx) or try our Teleport Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
+</Admonition>
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and `*.teleport.example.com`. You can substitute the address of your Teleport Proxy Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
+
+<Admonition type="note" title="Application Access and DNS">
+(!docs/pages/includes/dns-app-access.mdx!)
+</Admonition>
+
+## Step 1/5. Create an IAM role for DynamoDB access
+
+The first step is to create an IAM role with desired IAM permissions that you wish to grant to a Teleport user. In this example, you will create an IAM role with full DynamoDB access. **Skip this step** if you already have the roles you want to provide access to.
+
+Now go to the IAM -> Access Management ->
+[Roles](https://console.aws.amazon.com/iamv2/home#/roles), then press Create Role.
+
+Select the "AWS account" option which creates a default [trust
+policy](https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/)
+to allow other entities in this account to assume this role.
+
+![Create Role Step 1](../../../img/database-access/guides/dynamodb-create-role-1.png)
+
+Press Next. Find the AWS managed policy `AmazonDynamoDBFullAccess` by Filter and then select the policy.
+
+![Create Role Step 2](../../../img/database-access/guides/dynamodb-create-role-2.png)
+
+Press Next. Enter a role name and press Create role.
+
+![Create Role Step 3](../../../img/database-access/guides/dynamodb-create-role-3.png)
+
+## Step 2/5. Configure Teleport IAM role mapping
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a role with the `aws_role_arns` field listing all IAM
+role ARNs this particular role permits its users to assume:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: aws-dynamodb-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    aws_role_arns:
+    - arn:aws:iam::123456789000:role/ExampleTeleportDynamoDBRole
+```
+
+The `aws_role_arns` field supports template variables so they can be populated
```

Comment:
If this paragraph isn't necessary for following the instructions in this tutorial, I think we can either remove it, move the link to the "Next steps" section, or add it to a `Details` box.